certificates/generate-certificates.sh

```diff
@@ -3,6 +3,13 @@
 # Generates certificates for local development. A single certificate, that
 # is unique to you, is added to your browser
 
```

Comment:
would it be a good idea to do this near the top?

```suggestion
set -eu
```

Reply:
Added pipefail and reporting as well